docs/newContributors.md

```diff
@@ -0,0 +1,48 @@
+<h1 align="center">Hello Peeps 👋</h1>
+<h2 align="center">Welcome to the <a href="https://github.com/siv2r/kidney-exchange">kidney-exchange</a> Repo 🎉</h2>
+
+<h3 align="center">Here is, Complete Guide to Contribute to this Repo.</h3>
+
+<br>
+
+- ##### Firstly, Join our [Discord](https://discord.com/invite/d7j63WZV82) Server, Introduce yourself there, Interact with Others as 
+    > **Open Source is more about Collaboration.**
+
+- ##### So to start Contibuting to our Project you can Solve existing issues or if you have found any bug or  want to add new featurre to Project you can create New Issue , that's purely your choice.
+  ![](../images/Issues.png)
+
+- ##### Project Maintainers and Mentors Will Guide you With the issue and Incase of any problem you face ,you can take help from the community in the discord server.
```

Comment:
You have used capital letters in the middle of the document like in this line. Could you make them small and use them only to emphasise something.